app/javascript/modules/listings/BuyHeader.scss

```diff
@@ -28,4 +28,4 @@
   @apply mb-8;
   @apply text-3xl;
   @apply md:text-4xl;
-}
\ No newline at end of file
+}
```

Comment:
unsure what this change is about - [I'm seeing ](https://github.com/SFDigitalServices/sf-dahlia-web/blob/main/app/javascript/modules/listings/BuyHeader.scss#L31)that the main branch has that closing bracket.